shared/js/dataDriver.js

```diff
@@ -277,13 +292,15 @@ loop.DataDriver = function() {
      * @param {String} token The room token to use for connections.
      */
     _connectToRoom(token) {
+      this._retryTimeout = this.INITIAL_RETRY_TIMEOUT;
       this._roomToken = token;
       this._sse = new EventSource(this._buildUrl() + "?" + this._buildQuery({
         limitToLast: 1,
         orderBy: '"timestamp"'
       }));
       this._sse.addEventListener("open", this);
       this._sse.addEventListener("put", this);
+      this._sse.addEventListener("error", this);
```

Comment:
The current implementation doesn't seem like it would trigger exponential backoff as when it attempts to `_connect` again, it'll reset the delay back to initial. I'm also unclear if calling `addEventListener` with a new `_sse` but with the same callback/`this` will cause problems of receiving events from multiple `EventSource` instances.